frontend/src/components/module-editor/EditableContentBlock.tsx

```diff
@@ -55,6 +57,12 @@ const EditableContentBlock = ({
               </svg>
             </Box>
             {SelectContentBlock(block)}
+            <EditContentOptionsMenu
+              isVisible={isHovered}
+              onEditClick={() => console.log("edit")}
+              onCopyClick={() => console.log("copy")}
+              onDeleteClick={() => console.log("delete")}
```

Comment:
(nit, optional): We could change these to empty functions for now, since otherwise these warnings will show up on other people's PRs until the subsequent ticket is worked on.
```suggestion
              onDeleteClick={() => {}}
```